convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -163,6 +163,26 @@ def _enable_repos(self):
                 diagnosis="Loading repository metadata failed with error %s." % (str(e)),
             )
 
+    def _swap_problematic_packages(self):
+        """Swap system packages for their RHEL counterparts in the transaction.
+
+        Some packages need to be manually injected in the transaction as a
+        "swap", since those packages are not always able to be installed
+        automatically by yum if they don't exist in the system anymore, this
+        can cause problems during the transaction as missing dependencies.
+        """
+        # Related issue: https://issues.redhat.com/browse/RHELC-1130, see comments
+        # to get more proper description of solution
```

Comment:
```suggestion
        can cause problems during the transaction as missing dependencies.
        Related issue: https://issues.redhat.com/browse/RHELC-1130, see comments
        to get more proper description of solution
        """

Reply:
I discussed those with @r0x0d and we agreed on having the mention of the issue under the main comment, in the `#` section. https://github.com/oamg/convert2rhel/pull/1035#discussion_r1462116129

But I see I left there duplicates, I'll remove them